app/assets/javascripts/application.js

```diff
@@ -18,9 +18,6 @@
 //= require best_in_place
 //= require_tree .
 
-// @import "materialize";
-// @import "https://fonts.googleapis.com/icon?family=Material+Icons";
```

Comment:
unneeded per https://github.com/mkhairi/materialize-sass -- if it's included in `application.scss` we don't need it here!